src/components/Prescription/Prescription.css

```diff
@@ -61,12 +63,28 @@
     border-radius: 10px;
     margin-right: 40px;
 }
+
+.bottom-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100%;
+    width: 80%;
+}
   
 .script-message {
+    height: 50%;
     width: 100%;
-    padding: 20px;
+    /* padding: 20px; */
```

Comment:
Should we remove this commented-out line?

Reply:
Removed!